digital-twins/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure IoT Plug-and-Play Preview
+
+This folder contains code examples showing usage of the Azure IoT Plug-and-Play APIs and client libraries for Node.js.
+
+**Please note that these features are currently in preview, and as such, the final shape of the API might still change**
+
+The `Quickstarts` folder contains code examples associated with the quickstarts and how-to guides for Azure IoT Plug-and-Play.
+
+The `Samples` folder contains an assorted collection of simple samples showing usage of the various APIs and have their own readmes within each folder explaining how to use them.
```

Comment:
The `Samples` folder contains a collection of simple samples that show the usage of the various APIs. Each folder contains its own readme that explains how to use the sample.